### ISSUE TITLE
Currency locale is dependant on the user's environment

### ISSUE THREAD

**Issue by DF1229** [DF1229/money-tracker#12]
Currently, all currency format uses `interaction.locale`, which gets its value from the user.
This messes with the formatting, which is done through `Intl.NumberFormat`.

For example, formatting 123456.78 to Euro should end up looking like €123.456,78.
In the current situation users who have their client set to a locale like `en-GB` or `en-US`, but use the euro, get their currency formatted as €123,456.78.
In other words, in the style of the US dollar.
